SUB-PROPERTY PERMUTATIONS:.md

```diff
@@ -0,0 +1,243 @@
+SUB-PROPERTY PERMUTATIONS:
+
+Axiom: for [B] object_subProperty of [A], any {x,y} pair of instances in B will occur in [A].
+However, it is not the case that any {x,y} in A will occur in [B].
+So, the issue is to find where the logical conditions of Sub_Property [B] violate those of their parent object_property [A]. 
+
+For any Object Property [A] as (Functional, Inverse Functional, Transitive, Symmetric, Asymmetric, Reflexive, Irreflexive),
+Consider whether its sub_Property [B] violates what is maintained by the logical committments of [A].
+Cases are considered in piecemeal fashion examining all possible combinations of some [A] paired with each possible [B].
+Permutation involving inverses of [A] & [B] (A', B' respectively) are forthcoming. 
+
+1. [A] as Functional
```

Comment:
Consider using tables to organize combinations of role constraints under each type of property constraint